src/10-animation.md

```diff
@@ -23,7 +23,7 @@ Then we take the `player_sprite_index` and use that when specifying the `path` o
 
 ## A Note on Spritesheets
 
-If you've done any game development before, you may be familiar with spritesheets, where each frame of an animation is contained within one image file. When using a spritesheet for animation, instead of changing which image file path is used for the sprite to render, you change which piece of the large spritesheet you render. [The DragonRuby GTK docs have a detailed example of how to do this.](https://bit.ly/drgtk-spritesheet)
+If you've done any game development before, you may be familiar with spritesheets, where each frame of an animation is contained within one image file. When using a spritesheet for animation, instead of changing which image file path is used for the sprite to render, you change which piece of the large spritesheet you render. [The DragonRuby GTK docs have a detailed example of how to do this.](http://docs.dragonruby.org.s3-website-us-east-1.amazonaws.com/#----animation-using-sprite-sheet---main-rb)
```

Comment:
goooood catch :) and thanks very much for the fix.

It would probably be good to use the official domain instead of the direct link to the s3 file (we might eventually move it to another place and the link would break again)
(the bit.ly was there exactly for that reason because there was a problem with the domain IIRC) (in fact... we should change all links ;))
```suggestion
If you've done any game development before, you may be familiar with spritesheets, where each frame of an animation is contained within one image file. When using a spritesheet for animation, instead of changing which image file path is used for the sprite to render, you change which piece of the large spritesheet you render. [The DragonRuby GTK docs have a detailed example of how to do this.](http://docs.dragonruby.org/#----animation-using-sprite-sheet---main-rb)
```